.env.example

```diff
@@ -3,6 +3,13 @@ LOGGING_LEVEL=info
 
 WEB_CONCURRENCY=1
 
+V8_PROFILER_ENABLED=
```

Comment:
Nevermind, I thought all other variables in Gamibt are prefixed with `GAMBIT_` too.